upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -371,6 +371,36 @@ There you can find links to upgrade notes for other versions too.
         +        Assert::assertNotSame($errorIdFirstAccess, $errorIdSecondAccess);
         +    }
         ```
+- add possibility to override admin styles from project-base
+ ([#1472](https://github.com/shopsys/shopsys/pull/1472))
+    - delete all files from `project-base/src/Shopsys/ShopBundle/styles/admin/` and create two new files in it - `main.less` and `todo.less` (this file will be empty)
+
+    main.less file content:
+    ```diff
```

Comment:
You don't need to use `diff` here but I think that you could use `less` so it could be highlighted properly

Reply:
Less did not work. Used css and it highlight code properly. Thx for this note.